azkaban-common/src/main/java/azkaban/executor/ExecutionFlowDao.java

```diff
@@ -123,6 +123,16 @@ public void uploadExecutableFlow(final ExecutableFlow flow)
     }
   }
 
+  public List<Pair<ExecutionReference, ExecutableFlow>> fetchAgedQueuedFlows(final Duration minAge)
+      throws ExecutorManagerException {
+    try {
+      return this.dbOperator.query(FetchAgedQueuedExecutableFlows.FETCH_FLOWS_QUEUED_FOR_LONG_TIME,
+          new FetchQueuedExecutableFlows(), System.currentTimeMillis() - minAge.toMillis());
+    } catch (final SQLException e) {
+      throw new ExecutorManagerException("Error fetching active flows", e);
```

Comment:
Can the error message reflect the purpose of the query more closely? Something like "Error fetching executions queued for a long time"